tests/Behat/features/restore-to-draft.feature

```diff
@@ -0,0 +1,29 @@
+@javascript
+Feature: Restore to draft
+  As a CMS author
+  I want to restore archived version to draft
+
+  Background:
+    Given a "page" "Home"
+      And a "page" "MyPage"
+      And the "group" "EDITOR" has permissions "Access to 'Pages' section" and "Access to 'Archive' section"
+      And I am logged in as a member of "EDITOR" group
+      And I go to "/admin/pages"
+      And I should see "MyPage"
+      And I click on "MyPage" in the tree
+      And I wait 2 seconds
+      And I wait 2 seconds
```

Comment:
You shouldn't have to wait at all - but if you do, do it in a single command instead of two separate commands.

Reply:
Ha, I removed few steps between and skipped "waiting" :)